cmd/cryptogen/main.go

```diff
@@ -67,13 +68,17 @@ type UsersSpec struct {
 }
 
 type OrgSpec struct {
-	Name          string       `yaml:"Name"`
-	Domain        string       `yaml:"Domain"`
-	EnableNodeOUs bool         `yaml:"EnableNodeOUs"`
-	CA            NodeSpec     `yaml:"CA"`
-	Template      NodeTemplate `yaml:"Template"`
-	Specs         []NodeSpec   `yaml:"Specs"`
-	Users         UsersSpec    `yaml:"Users"`
+	Name               string        `yaml:"Name"`
+	Domain             string        `yaml:"Domain"`
+	EnableNodeOUs      bool          `yaml:"EnableNodeOUs"`
+	CACertExpiry       time.Duration `yaml:"CACertExpiry"`
+	IdentityCertExpiry time.Duration `yaml:"IdentityCertExpiry"`
+	TLSCertExpiry      time.Duration `yaml:"TLSCertExpiry"`
```

Comment:
We need integration tests to ensure that the durations are properly handled when present and missing. The unit tests at the CA level do not ensure that the tool works as intended. Unfortunately, `cryptogen` does not seem to have integration tests so there's not much to build on.